Signal/src/util/OWSBackupExportJob.m

```diff
@@ -147,26 +149,29 @@ - (void)start
         [self updateProgressWithDescription:NSLocalizedString(@"BACKUP_EXPORT_PHASE_EXPORT",
                                                 @"Indicates that the backup export data is being exported.")
                                    progress:nil];
-        if (![self exportDatabase]) {
-            [self failWithErrorDescription:
-                      NSLocalizedString(@"BACKUP_EXPORT_ERROR_COULD_NOT_EXPORT",
-                          @"Error indicating the a backup export could not export the user's data.")];
-            return;
-        }
-        if (self.isComplete) {
-            return;
-        }
-        [self saveToCloud:^(NSError *_Nullable saveError) {
-            if (saveError) {
-                [weakSelf failWithError:saveError];
+        [self exportDatabase:^(BOOL exportDatabaseSuccess) {
```

Comment:
Yes, I'm going to finally address this in the 4th PR.  I keep putting it off to avoid rebase hell because this work is rife with completions, and the work is always ahead of the PR process.